owenhallgren/stretch-project
# Feature/styling app

## What does this PR do?

-  This changes the styling of our home page, cards, and data tables

## How should it be tested in the UI?

- node start the api
- npm start the front end react app
- check and confirm styling changes are there and looks correct
